prov/cxi/include/cxip.h

```diff
@@ -138,7 +138,7 @@
 	(FI_SOURCE | FI_SOURCE_ERR | FI_LOCAL_COMM | \
 	 FI_REMOTE_COMM | FI_RMA_EVENT | FI_MULTI_RECV | FI_FENCE | FI_TRIGGER)
 #define CXIP_EP_CAPS (CXIP_EP_PRI_CAPS | CXIP_EP_SEC_CAPS)
-#define CXIP_DOM_CAPS (FI_LOCAL_COMM | FI_REMOTE_COMM | FI_AV_USER_ID)
+#define CXIP_DOM_CAPS (FI_LOCAL_COMM | FI_REMOTE_COMM | FI_AV_USER_ID | FI_PEER)
```

Comment:
I think to fully support FI_PEER, you need to support counters too. I know link doesn't use them so it's not the biggest of deals but it probably wouldn't be too hard - how hard do you think it would be to add peer counter support?

Reply:
I have to admit, I haven't looked at counters very closely, so I don't know how much work to add it in this PR. I would prefer to have a separate PR for it.